app/src/main/java/blockeq/com/stellarwallet/activities/PinActivity.kt

```diff
@@ -189,7 +190,7 @@ class PinActivity : AppCompatActivity(), PinLockListener {
 
     //region Generate Stellar Account
     private fun generateStellarAddress(mnemonic : String) {
-        val keyPair = Wallet.createKeyPair(mnemonic.toCharArray(), null, 0)
+        val keyPair = SupportedMnemonic.createKeyPair(mnemonic.toCharArray(), null, 0)
```

Comment:
What is 0? Better to create a constant called USER_INDEX = 0 so it describes it better.